src/lib.rs

```diff
@@ -26,3 +26,27 @@ extern crate pretty_assertions;
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
 pub use server::LspServer;
+
+#[macro_export]
+macro_rules! walk_ast_package {
+    ($visitor:expr, $package:ident) => {{
+        let mut visitor = $visitor;
+        flux::ast::walk::walk(
+            &mut visitor,
+            flux::ast::walk::Node::Package(&$package),
+        );
+        visitor
+    }};
+}
+
+#[macro_export]
+macro_rules! walk_semantic_package {
+    ($visitor:expr, $package:ident) => {{
+        let mut visitor_instance = $visitor;
+        flux::semantic::walk::walk(
+            &mut visitor_instance,
+            flux::semantic::walk::Node::Package(&$package),
```

Comment:
No, because we're referencing two different things in the `flux::semantic::walk` module: the `walk` function we're actually calling, and the `Node::Package` enum variant.